itl/command-plugins.conf

```diff
@@ -502,7 +502,7 @@ object CheckCommand "http" {
 		}
 		"-k" = {
 			value = "$http_header$"
-			description = "Any other tags to be sent in http header"
+			description = "Any other tags to be sent in http header (may be repeated)"
```

Comment:
... e.g. from here?

Reply:
That's how I formulated it in the first PR. I took it from your answer "Please have a look how the other commands describe it." that you weren't satisfied with it (and I agree, it could be clearer).

Reply:
Ah! Sorry!

Reply:
So the other commands do it like you did here?

Reply:
As I wrote above, I could not make out a "standard phrasing" for the fact that a parameter accepts arrays. In the command definitions sometimes it's "may be repeated", sometimes "may be used repeatedly".

In the documentation file the wording is even more inconsistent (just a few examples):
```
doc/10-icinga-template-library.md:1208:smtp_command          | **Optional.** SMTP command (may be used repeatedly).
doc/10-icinga-template-library.md:1209:smtp_response         | **Optional.** Expected response to command (may be used repeatedly).
doc/10-icinga-template-library.md:206:apt_include             | **Optional.** Include only packages matching REGEXP. Can be specified multiple times the values will be combined together.
doc/10-icinga-template-library.md:207:apt_exclude             | **Optional.** Exclude packages matching REGEXP from the list of packages that would otherwise be included. Can be specified multiple times.
doc/10-icinga-template-library.md:240:by_ssh_command     | **Required.** The command that should be executed. Can be an array if multiple arguments should be passed to `check_by_ssh`.
doc/10-icinga-template-library.md:356:disk\_ignore\_eregi\_path | **Optional.** Regular expression to ignore selected path/partition (case insensitive). Multiple regular expression strings must be defined as array.
doc/10-icinga-template-library.md:1937:nscp_disk_exclude      | **Optional.** Drive character, default to none. Can be an array of drive characters if multiple drives should be excluded.
```
But generally, the documentation is  a bit more verbose than the descriptions from the command definition.